.circleci/meta-linter/main.go

```diff
@@ -18,21 +18,104 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"bitbucket.org/creachadair/stringset"
 )
 
 func main() {
 	errors := []error{}
 
-	// Run each checker and run up the list of errors.
-	errors = append(errors, checkRulesRegistered()...)
-	errors = append(errors, ruleFilename()...)
-	errors = append(errors, checkRulesDocumented()...)
+	// Keep track of rules processed, and which ones have one or more failures.
+	pass := stringset.New()
```

Comment:
Consider to give `pass` and `fail` more informative names, like `passedRules` and `failedRules`.

Reply:
Done.